### PR TITLE
Updates to doc for MSVC construction vars

### PR DESCRIPTION
**Pull request by mwichmann**
Some markup and minor wording changes for various MSVC tool variables.

Signed-off-by: Mats Wichmann <mats@linux.com>

## Contributor Checklist:

* [ ] I have created a new test or updated the unit tests to cover the new/changed functionality.
* [ ] I have updated `CHANGES.txt` (and read the `README.rst`)
* [X] I have updated the appropriate documentation

### CODE REVIEW

**Comment by mwichmann** [comment]
Hmmm... @bdbaddog this seems to have skipped travis as well, even though the commit only had `[skip appveyor]`

_Update_: it was just delayed, build happened. Will fix.

**Comment by mwichmann** [comment]
Looks like maybe the build should abort more aggressively?  Snip from build log:
```
$ python bin/docs-update-generated.py
Generating entity files...
error in SCons/Tool/msvc.xml
Traceback (most recent call last):
  File "bin/scons-proc.py", line 401, in <module>
    h = parse_docs(args, include_entities=False)
  File "bin/scons-proc.py", line 73, in parse_docs
    h.parseContent(content, include_entities)
  File "/home/travis/build/SCons/scons/bin/SConsDoc.py", line 583, in parseContent
    t.parseContent(content, include_entities)
  File "/home/travis/build/SCons/scons/bin/SConsDoc.py", line 396, in parseContent
    self.root = etree.fromstring(content)
  File "src/lxml/etree.pyx", line 3235, in lxml.etree.fromstring
  File "src/lxml/parser.pxi", line 1876, in lxml.etree._parseMemoryDocument
  File "src/lxml/parser.pxi", line 1757, in lxml.etree._parseDoc
  File "src/lxml/parser.pxi", line 1068, in lxml.etree._BaseParser._parseUnicodeDoc
  File "src/lxml/parser.pxi", line 601, in lxml.etree._ParserContext._handleParseResultDoc
  File "src/lxml/parser.pxi", line 711, in lxml.etree._handleParseResult
  File "src/lxml/parser.pxi", line 640, in lxml.etree._raiseParseError
  File "<string>", line 386
lxml.etree.XMLSyntaxError: Opening and ending tag mismatch: envar line 383 and para, line 386, column 8
Generation failed
The command "python bin/docs-update-generated.py" exited with 0.
3.04s$ python bin/docs-validate.py
0.97% (1/103) doc/design/acks.xml
...
```
(IMO) we should stop at the "Generation failed" point, not continue - although actually this is noted in a comment I myself left in `docs-update-generated`:
```
        # No-op: scons-proc doesn't actually set an exit code at the moment.
        if cp.returncode:
            print("Generation failed", file=sys.stderr)
```

**Comment by bdbaddog** [comment]
@mwichmann - agree. if any of those scripts fail the exit status should indicate..

**Comment by mwichmann** [comment]
I hacked it quickly and can now indicate the failure (artificially introduced syntax error in the xml - not that *that* takes much work!), but I lost the contents of the traceback, so I have a little more digging to do on that part.  Separate PR for get the exit status working OK?

File: SCons/Tool/msvc.xml [hunk 1]
```diff
@@ -438,13 +453,13 @@ Valid values for Windows are
...
-(for 32 bits); 
+(for 32 bit);
 <literal>amd64</literal>,
 <literal>arm64</literal>,
 <literal>emt64</literal>,
 <literal>x86_64</literal>
-(for 64 bits);
-and <literal>ia64</literal> (Itanium).
+(for 64 bit);
+and <literal>ia64</literal> (Itanium) (deprecated).
```
Comment (bdbaddog) [inline SCons/Tool/msvc.xml]: Why add deprecated?
SCons will support that as long as we support a MSVC compiler toolchain which can target itanium?

**Comment by mwichmann** [comment]
The fact Microsoft has not supported Itanium since VS2010 is enough to consider it deprecated, no?  I could annotate that and the two arm arches with the version they do and/or don't apply to if that's better.

**Comment by bdbaddog** [comment]
Maybe just a note to say that not all target archs are supported by all MSVC versions. Please check your compiler install to verify.

**Comment by mwichmann** [comment]
sure, I can do that, easier than digging out the MS data.

**Comment by bdbaddog** [comment]
Yup.. it's an exercise left for the reader.. ;)

**Pull request merged**